configs/openmrs/initializer_config/encountertypes/encountertypes_forms.csv

```diff
@@ -0,0 +1,19 @@
+Uuid,Void/Retire,Name,display:en,display:fr,Description,View privilege,Edit privilege,_order:1000,
```

Comment:
I'd simply add these to the `encountertypes.csv` file.

Reply:
The intention was to separate concerns because the other encounters were not working for the current setup

Reply:
> The intention was to separate concerns because the other encounters were not working for the current setup

Not sure I understand. I meant combining all encounter types in one file. And you are implying that this is not possibly working?

Reply:
i.e. `encountertypes_forms.csv` ➔ `encountertypes.csv`...

Reply:
@Ruhanga please have a look now